index.bs

```diff
@@ -2557,77 +2467,46 @@ partial interface MLGraphBuilder {
   </div>
 </details>
 
-<details open>
-  <summary>
-    The element-wise unary operation algorithms invoke the [=MLGraphBuilder/element-wise-unary-op | create element-wise unary operation=] steps as follows.
-  </summary>
-  <div class=algorithm-steps>
-    <div algorithm>
-    The <dfn method for=MLGraphBuilder>abs(|input|)</dfn> method steps are:
-        1. Let |output| be the result of running the [=MLGraphBuilder/element-wise-unary-op | create element-wise unary operation=] given "abs" and |input|.
-            1. If that [=exception/throws=] an error, then re-[=exception/throw=] the error.
-        1. Return |output|.
-    </div>
+#### abs #### {#api-mlgraphbuilder-unary-abs}
+Compute the absolute value of the input tensor, element-wise. See [[#api-mlgraphbuilder-unary]] for more detail.
```

Comment:
This doesn't reference an algorithm for the construction, and "abs" is never passed. 

If you want a more compact form, you can write something like:

> The abs(input) method steps are to return the results of running create a unary operation given "abs" and input. Rethrow any exceptions.

Reply:
Also, and not new in this PR, but the description of the behavior (i.e. "Compute the absolute value of the input tensor, element-wise") is written normatively but isn't connected to anything. The spec is going to need to define when it executes.

One approach is to encapsulate that as an algorithm that are associated with the created MLOperand (like a lambda) and define the inference-time behavior.

As noted, this lack of precision isn't new in this PR. But this PR **_does_** move things around substantially in a way that I don't think actually improves the rigor of the spec, so maybe don't try to do it at the same time as introducing the new ops?

Reply:
Part of this change does attempt to make the operator section in the table of content less cluttered and easier to read and discover. 

To your earlier feedback on the `abs` section, it was the attempt to prop up the list of all the element-wise operators in the table of content so it can be more easily discovered. Since they all more or less share the same semantic, the description that is unique to each individual operator unfortunately becomes a one-liner, which I can understand how it may not be desirable to some. 

There is no good way to record all the many similar element-wise operators in the spec without also making a lot of redundant text that makes the spec prone to inconsistency over time. For this spec, we have historically chosen to put them all into a sub-section e.g. "element-wise binary operations", etc. If you have other suggestions, please feel free.

Reply:
Fixed.

Reply:
Fixed in https://github.com/webmachinelearning/webnn/pull/478/commits/873eec1e656253ed5e5f600f0ebf7e5fd12b9746